Jellyfin.Server.Implementations/Trickplay/TrickplayManager.cs

```diff
@@ -498,6 +498,11 @@ public async Task<Dictionary<string, Dictionary<int, TrickplayInfo>>> GetTrickpl
         var trickplayManifest = new Dictionary<string, Dictionary<int, TrickplayInfo>>();
         foreach (var mediaSource in item.GetMediaSources(false))
         {
+            if (mediaSource.IsRemote)
+            {
+                continue;
+            }
+
             var mediaSourceId = Guid.Parse(mediaSource.Id);
```

Comment:
```suggestion
            if (mediaSource.IsRemote || !Guid.TryParse(mediaSource.Id, out var mediaSourceId)
            {
                continue;
            }
```

Reply:
Do we really want to do this?

Reply:
dont see harm in just checking for it.
Could add a log debug entry as well but its better then just failing.

Reply:
better would be to make the Id field non nullable/required but thats a bit outside the scope